src/quick-lint-js/diag/diagnostic-types-2.h

```diff
@@ -3599,6 +3599,15 @@ struct Diag_Using_Dot_After_Optional_Chaining {
   Source_Code_Span dot_op;
   Source_Code_Span optional_chain_op;
 };
+
+struct Diag_Unexpected_Await_On_Function_Declaration {
+  [[qljs::diag("E0719", Diagnostic_Severity::warning)]]  //
+  // clang-format off
+  [[qljs::message("Unexpected 'await' keyword on function declaration. Maybe you meant 'async'?",
+                  ARG(await_keyword))]]  //
```

Comment:
You should follow the syntactic style of existing diagnostics (like E0178) instead:
```suggestion
  [[qljs::message("unexpected 'await' keyword on function declaration; maybe you meant 'async'?",
                  ARG(await_keyword))]]  //
```